code/datums/weather/weather_types/ash_storm.dm

```diff
@@ -72,10 +71,6 @@ GLOBAL_LIST_EMPTY(ash_storm_sounds)
 			var/thermal_protection = H.easy_thermal_protection()
 			if(thermal_protection >= FIRE_IMMUNITY_MAX_TEMP_PROTECT)
 				return TRUE
-		if(isliving(L))// if we're a non immune mob inside an immune mob we have to reconsider if that mob is immune to protect ourselves
-			var/mob/living/the_mob = L
-			if("ash" in the_mob.weather_immunities)
-				return TRUE
```

Comment:
pretty sure you need to replace this with the HAS_TRAIT check, rather than just removing it

Reply:
i'm pretty sure this is in reference to like
vore
or maybe it's shit like bags of holding but if it's vore tbqh i'm honestly fine with this line eating shit i'll test it later